### PR TITLE
Activate update tests

### PR DESCRIPTION
**Pull request by EmilyBourne**
All `update` tests seem to be working already as `update` is simply a multi-element version of `union`. This PR simply activates the existing tests and updates the documentation

### CODE REVIEW

File: tests/epyccel/test_epyccel_sets.py [hunk 1]
```diff
@@ -178,14 +178,14 @@ def Discard_wrong_arg():
     python_result = Discard_wrong_arg()
     assert python_result == pyccel_result
 
-def test_update_basic(python_only_language):
+def test_update_basic(language):
```
Comment (jalalium) [inline tests/epyccel/test_epyccel_sets.py:181]: I think we should support `update` with more than one argument. Is this a choice that has been made?

Comment (EmilyBourne) [reply tests/epyccel/test_epyccel_sets.py:181]: Ah good point, I thought there was something I was supposed to do before marking this as ready. I just couldn't remember what!

Comment (EmilyBourne) [reply tests/epyccel/test_epyccel_sets.py:182]: Fixed : c98b12d

Test added : 5238c30

File: pyccel/parser/semantic.py [hunk 2]
```diff
@@ -5323,36 +5323,42 @@ def _build_SetUpdate(self, expr, args):
             CodeBlock or For containing SetAdd objects.
         """
         if isinstance(expr, DottedName):
-            iterable = expr.name[1].args[0].value
+            iterable_args = [a.value for a in expr.name[1].args]
             set_obj = expr.name[0]
         elif isinstance(expr, AugAssign):
-            iterable = expr.rhs
+            iterable = [expr.rhs]
```
Comment (EmilyBourne) [inline pyccel/parser/semantic.py:5329]:
```suggestion
            iterable_args = [expr.rhs]
```

**Review by jalalium — APPROVED** [review]
Well done!

**Pull request merged**